doc/nrf/libraries/dfu/dfu_target.rst

```diff
@@ -103,6 +104,67 @@ This DFU target downloads the serialized modem firmware to an external flash mem
 Once the modem firmware has been downloaded, the application should use :ref:`lib_fmfu_fdev` to write the firmware to the modem.
 The DFU target library does not perform the upgrade and calling the :c:func:`dfu_target_schedule_update` function has no effect.
 
+.. _lib_dfu_target_suit_style_update:
+
+SUIT-style upgrades
+-------------------
+
+SUIT-style firmware upgrades can be used for updates to the SUIT envelope and cache images.
+Depending on the image number, this type of DFU writes the data provided to the :c:func:`dfu_target_write` function into the following partitions:
+
+* Image 0: `dfu_partition`
+* Image 1: Reserved by SUIT to store the firmware included in the SUIT envelope.
+* Image 2..n: `dfu_cache_partition_n`
+
+Before calling the :c:func:`dfu_target_init` and :c:func:`dfu_target_write` functions, the application must call the :c:func:`dfu_target_suit_set_buf` function to allocate the buffer used during the firmware update process.
+The buffer size must be at least the size of the largest chunk that will be downloaded at a single time.
+The buffer will be used for processing all images, so there is no need to allocate a new buffer for each image.
+
+You can upgrade your device in several ways:
+
+.. tabs::
+
+   .. tab:: Single image
+
+      Upgrade the device using the SUIT envelope that contains the manifests and firmware.
+
+      The SUIT manifests and firmware will be stored in the `dfu_partition` partition.
+      After that, the `dfu_cache_partition_0` partition will be created automatically just after the `dfu_partition` partition and will contain the firmware.
+
+   .. tab:: Cache processing
+
+      Upgrade the device using the SUIT envelope that contains the manifests only, while the firmware is stored in the cache images.
+
+      The :kconfig:option:`DFU_TARGET_SUIT_CACHE_PROCESSING` Kconfig option must be enabled.
+      The SUIT manifests will be stored in the `dfu_partition` partition.
+      The firmware will be stored in the `dfu_cache_partition_1` partition.
+      The `dfu_cache_partition_0` partition is reserved by SUIT and should not be used in this case.
```

Comment:
Is there anything preventing us from using it?

Reply:
Actually no, but this is a case for multi-image. I will remove this line.